javascript-data-model/order-history.js

```diff
@@ -0,0 +1,63 @@
+var orderHistory = [
+  {
+    OrderPlaced: 'August 4, 2020',
+    items: [
```

Comment:
so this is moving in the right direction, what we want to update it to do is contain only the individual product data in the `items` objects and keep the overall order data in the order object: 
so the thing you want to add to this data model is a property `items` which will contain an array of objects in which each object contains an individual items data: 
```javascript
var orderHis = [
  {
    orderedDate: '2020-08-04',
    totalPrice: 34,
    shipTo: 'JS Masher',
    orderNum: '114-3941689-8772232',
    deliveredDate: '2020-08-08',
    orderedDate: '2-15-21',
    items:[{
      type: "book",
      title: "A New Way to Learn"
    },
    {
      type: "electronics",
      title: "Blu Ray Player"
    },
    {
      type: "book",
      title: "How to forget"
    },
    {
      type: "book",
      title: "how to remember"
    }
  ]
  }
]
```